addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6295,6 +6309,20 @@ registry.ImageTools = ImageHandlerOption.extend({
 
         const svg = new DOMParser().parseFromString(svgText, 'image/svg+xml').documentElement;
         const svgAspectRatio = parseInt(svg.getAttribute('width')) / parseInt(svg.getAttribute('height'));
+
+        // Apply the right animation duration if there is an element animate
+
+        const svgAnimateElement = svg.querySelector('animate');
+        if (svgAnimateElement) {
+            if (!img.dataset.animationDuration) {
+                img.dataset.animationDuration = parseInt(svgAnimateElement.getAttribute('dur'));
+            } else {
+                svgAnimateElement.setAttribute('dur', img.dataset.animationDuration + 's');
```

Comment:
Don't concatenate strings like that, use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead